backend/src/ml_space_lambda/app_configuration/lambda_functions.py

```diff
@@ -18,39 +18,234 @@
 import logging
 import time
 
+import boto3
+
 from ml_space_lambda.data_access_objects.app_configuration import AppConfigurationDAO, AppConfigurationModel, SettingsModel
-from ml_space_lambda.utils.common_functions import api_wrapper
+from ml_space_lambda.data_access_objects.resource_metadata import ResourceMetadataDAO
+from ml_space_lambda.enums import EnvVariable, IAMEffect, IAMStatementProperty, ResourceType, ServiceType
+from ml_space_lambda.utils.account_utils import get_account_arn
+from ml_space_lambda.utils.common_functions import (
+    api_wrapper,
+    event_wrapper,
+    generate_exception_response,
+    generate_html_response,
+    retry_config,
+)
+from ml_space_lambda.utils.iam_manager import IAM_RESOURCE_PREFIX, IAMManager
+from ml_space_lambda.utils.mlspace_config import get_environment_variables
+from ml_space_lambda.utils.resoruce_utils import suspend_all_of_type
 
+iam_manager = IAMManager()
 log = logging.getLogger(__name__)
 app_configuration_dao = AppConfigurationDAO()
+env_variables = get_environment_variables()
+translate_client = boto3.client("translate", config=retry_config)
+resource_metadata_dao = ResourceMetadataDAO()
 
+service_group_disable_permissions = [
+    {
+        "Description": "Permissions shared by translate services",
+        "ServiceGroup": [ServiceType.BATCH_TRANSLATE, ServiceType.REALTIME_TRANSLATE],
+        "Statements": [
+            {
+                IAMStatementProperty.EFFECT: IAMEffect.DENY,
+                IAMStatementProperty.ACTION: [
+                    "translate:ListTerminologies",
+                    "translate:ListLanguages",
+                    "comprehend:Detect*",
+                    "comprehend:BatchDetect*",
+                ],
+                IAMStatementProperty.RESOURCE: "*",
+            },
+            {
+                IAMStatementProperty.EFFECT: IAMEffect.DENY,
+                IAMStatementProperty.ACTION: "iam:PassRole",
+                IAMStatementProperty.RESOURCE: get_account_arn("iam", f"role/{IAM_RESOURCE_PREFIX}*"),
+                IAMStatementProperty.CONDITION: {"StringEquals": {"iam:PassedToReception": "translate.amazonaws.com"}},
+            },
+        ],
+    }
+]
```

Comment:
This (and a lot of this file) seems like a lot to push into this file. I [followed your recommendation in my PR](https://github.com/awslabs/mlspace/pull/118/files#diff-ad040d66f148a6020ddc56f75bb1b76358c37a3a0ed2feb2e456f45916ea56a7) and split it out into a helper class so this file isn't so cluttered. I suggest you do the same.